src/utils/Dom.ts

```diff
@@ -733,6 +733,22 @@ export class Dom {
     return $$(<HTMLElement>this.el.cloneNode(deep));
   }
 
+  /**
+   * Determine if an element support a particular native DOM event.
+   * @param eventName The event to evaluate. Eg: touchstart, touchend, click, scroll.
+   */
+  public supportsEvent(eventName: string): boolean {
```

Comment:
Slightly disagree..

I appended the `on` inside the function, since naturally what I would want to check is the "pure" event name versus the event attribute. And so that seems like a better or more natural API to use.

For example, it seems more natural that you wanna check `is the event touchstart supported or not ? is the event scroll supported or not ?`

Versus : `is the property ontouchstart defined on the element or not ?` Because one is the actual event name, the other is a property on the object. 

So it's checking for the support of an event using the property name as a way to verify that.

Does it makes sense like this ?

Reply:
Ok, that's a good point. The other names I was playing in the case of a look-up by event name were:

```
hasHandlerForEvent(eventName:string) {}

canHandleEvent(eventName:string) {}
```

but I now think "supports" is good too.